internals/feature_helpers.py

```diff
@@ -58,6 +58,40 @@ def _get_future_results(async_features: Future | None) -> list[FeatureEntry]:
     return []
   return async_features.result()
 
+def get_features_in_release_notes(milestone: int):
+  cache_key = '%s|%s|%s' % (
+      FeatureEntry.DEFAULT_CACHE_KEY, 'release_notes_milestone', milestone)
+
+  cached_features = rediscache.get(cache_key)
+  if cached_features:
+    return cached_features
+
+  stages = Stage.query(
```

Comment:
Note: this will require a new index in index.yaml.  When we test on staging, the error message will tell us exactly how to define the new index.